src/pages/try.js

```diff
@@ -58,6 +70,17 @@ export default class Try extends Component {
     autoEvaluate: true,
     output: [],
   }
+
+  handleClick = ()=> {
+    var val = this.encodeBase64(this.state.reason);
+    window.location = val;
```

Comment:
This will cause an unnecessary refresh of the page. We should use `window.history.replaceState`